crates/oxc_sourcemap/src/encode.rs

```diff
@@ -25,9 +25,9 @@ pub fn encode(sourcemap: &SourceMap) -> JSONSourceMap {
     }
 }
 
-// Here using `serde_json::to_string` to serialization `names/source_contents/sources`.
+// Here using `v_jsonescape::b_escape` to serialize `names` / `source_contents` / `sources`.
```

Comment:
Ha ha ha back again!

```suggestion
// Here using `serde_json` to serialize `names` / `source_contents` / `sources`.
```